### ISSUE TITLE
ARIA live regions for showing errors with ActiveCode

### ISSUE THREAD

**Issue by sshumak** [RunestoneInteractive/rs#626]
When using Runestone with a screen reader, it can be difficult to know what errors there are with code that is ran with ActiveCode, as the errors show up below the code with little indication. One possible solution is to put the error messages in [ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions), as these would alert a screen reader that content has changed.

**Comment by bnmnetp** [comment]
I have tried to do that, but so far I have been unsuccessful in getting aria-live regions to work for the errors.  Doesn't seem like it should be hard, but so far no luck.

**Comment by bnmnetp** [comment]
working now in development, live this weekend.
